client/client_pool/include/client/client_pool/concord_client_pool.hpp

```diff
@@ -168,6 +168,8 @@ class ConcordClientPool {
                          const concord::config_pool::ConcordClientPoolConfig&);
   void CreatePool(concord::config_pool::ConcordClientPoolConfig&);
 
+  void AddMissingReconfigurationFields(std::vector<uint8_t>& request, const ClientPtr& chosenClient);
```

Comment:
Can we maybe state what we add? Is it the `signature`?

Reply:
Its sender id and signature. So this method represents all those which need to be populated when the client is decided by CLient Pool.

Reply:
I see. We could still name it such that it indicates the fields it adds.

Reply:
Thanks @dartdart26 , I changed the function name to AddSenderAndSignature.